src/main/scala/viper/gobra/ast/internal/PrettyPrinter.scala

```diff
@@ -650,7 +653,7 @@ class ShortPrettyPrinter extends DefaultPrettyPrinter {
     case Seqn(_) => emptyDoc
     case Label(label) => showProxy(label)
     case If(cond, _, _) => "if" <> parens(showExpr(cond)) <+> "{...}" <+> "else" <+> "{...}"
-    case While(cond, invs, measure, _) => "while" <> parens(showExpr(cond)) <> line <>
+    case While(cond, invs, measure, _, label) => opt(label)(text) <> opt(label.map(_ => ": "))(text) <> "while" <> parens(showExpr(cond)) <> line <>
```

Comment:
couldn't this be simplify as follows?
```suggestion
    case While(cond, invs, measure, _, label) => opt(label.map(l => s"${l}: "))(text) <> "while" <> parens(showExpr(cond)) <> line <>
```